install.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+
+set -u
```

Comment:
What does it mean?

Reply:
Treat unset variables as an error when substituting. Maybe `set -e` suits us better

Reply:
I think so too 🙂